Localization/TellMeWhen-enUS.lua

```diff
@@ -1093,6 +1093,10 @@ L["UIPANEL_FONT_JUSTIFY"] = "Horizontal Justification"
 L["UIPANEL_FONT_JUSTIFY_DESC"] = "Set the horizontal justification (Left/Center/Right) for this text display."
 L["UIPANEL_FONT_JUSTIFYV"] = "Vertical Justification"
 L["UIPANEL_FONT_JUSTIFYV_DESC"] = "Set the vertical justification (Top/Center/Bottom) for this text display."
+L["UIPANEL_GROUP_SIZE_X"] = "Item Width"
+L["UIPANEL_GROUP_SIZE_X_DESC"] = "Modifies the width of icons in this group."
+L["UIPANEL_GROUP_SIZE_Y"] = "Item Height"
+L["UIPANEL_GROUP_SIZE_Y_DESC"] = "Modifies the height of icons in this group."
```

Comment:
was difficult to figure out naming/position of l18n. Please let me know if I should use a different naming system.

Also - I used "Item Width/Height" since it modifies the items in the group, not the group itself.